### PR TITLE
Feature/change names

### PR DESCRIPTION
**Pull request by wvmitchell**
Updated Mary to Walt throughout. I love that Walt has a snake.

- Difficulties
- Specific things that I would like reviewed
- Questions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-Mary had a little snake,  
+Walt had a little snake,  
    Its scales were green as leaves,  
-And every where that Mary went  
+And every where that Walt went
```
Comment (letakeane) [inline README.md:3]: Walt! I love this.

File: README.md [hunk 2]
```diff
@@ -10,15 +10,15 @@ It made the children laugh and play,
 And so the Teacher turned him out,  
    But still he slithered near,  
 And waited patiently about,  
-   Till Mary did appear ;  
+   Till Walt did appear ;
```
Comment (letakeane) [inline README.md:13]: These weird spaces before punctuation are bothering me. Let's fix these.

**Review by letakeane — COMMENTED** [review]
Looks great, I love this change.

File: README.md (hunk 2 above)
Comment (wvmitchell) [reply README.md:13]: I agree, I'll update the PR

File: README.md [hunk 3]
```diff
@@ -10,15 +10,15 @@ It made the children laugh and play,
...
-   Till Mary did appear ;  
+   Till Walt did appear ;  
 And then he slid to her, and coiled  
    His body 'round her arm,  
 As if he said—‘ I’m not afraid—  
    You’ll keep me from all harm.’  
 
-‘ What makes the snake love Mary so ?’  
+‘ What makes the snake love Walt so ?’  
    The eager children cry—
```
Comment (wvmitchell) [inline README.md:20]: Actually, it seems the author intended the whitespace artistically...

Comment (letakeane) [reply README.md:20]: Ohhhhh. Okay. it's still gonna bother me but I'm cool with it!

**Comment by letakeane** [comment]
i'll merge!

**Pull request merged**